### PR TITLE
SUPESC-606 Add note about Availability Notification deploy file configuration

### PR DESCRIPTION
**Pull request by artem-png**
Ticket: https://spryker.atlassian.net/browse/CC-26425

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 1]
```diff
@@ -282,6 +282,25 @@ $config[AvailabilityNotificationConstants::STORE_TO_YVES_HOST_MAPPING] = [
 ];
 ```
 
+To enable the new endpoint, adjust a Deploy file:
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:285]: We are not enabling a new endpoint.

**Review by dmiseev — DISMISSED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md [hunk 2]
```diff
@@ -282,6 +282,25 @@ $config[AvailabilityNotificationConstants::STORE_TO_YVES_HOST_MAPPING] = [
 ];
 ```
 
+2.To set YVES host environment variables, adjust a Deploy file:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:285]: Use the real Deploy file name.
```suggestion
2. To set YVES host environment variables, adjust the `deply.*.yml` file:
```

Comment (artem-png) [reply _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:285]: We have many deploy files. Also customer can have it's own deploy file names

Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-availability-notification-feature.md:285]:
```suggestion
2.To set YVES host environment variables, adjust a deploy file:
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**